### ISSUE TITLE
Add the example/sample to demo the Cross-LAN E2E Communication features?

### ISSUE THREAD

**Issue by FengGaoCSC** [kubeedge/edgemesh#508]
**What would you like to be added/modified**:

- the example part: https://github.com/kubeedge/edgemesh#examples section only include [Cross-edge-cloud communication](https://edgemesh.netlify.app/guide/test-case.html#cross-edge-cloud)  example for **cross-edge-cloud** example.
- Could you add the example to demo the **Cross-LAN E2E Communication** features? I am very interesting about this example. 

**Why is this needed**:
Since I need this feature be demonstrated as a sample to use in our solution.

thanks very much.

**Comment by Poorunga** [comment]
You mean, do you need us to add test code for Cross-LAN Communication in E2E?

**Comment by FengGaoCSC** [comment]
> You mean, do you need us to add test code for Cross-LAN Communication in E2E?

No, what I expected is like the test case for end user  that same with https://edgemesh.netlify.app/guide/test-case.html#cross-edge-cloud

thanks very much
